ha/core/error.py

```diff
@@ -261,3 +248,13 @@ class HaSystemHealthException(SystemHealthError):
     Exception to indicate that some error happened during HA System Health processing.
     """
     pass
+
+class EventAnalyzer(HAError):
```

Comment:
Suffix it with Error or Exception.

Reply:
EventAnalyzerError  this already existed 
got duplicated removed this code